src/centos/scripts/start-marklogic.sh

```diff
@@ -129,6 +132,14 @@ else
     ML_ADMIN_USERNAME="${MARKLOGIC_ADMIN_USERNAME}"
 fi
 
+if [[ -f "$SECRET_WALLET_PWD_FILE" ]] && [[ -n "$(<"$SECRET_WALLET_PWD_FILE")" ]]; then
+    log "Using docker secret for wallet-password."
```

Comment:
log "Using Docker secret for wallet-password."

(capitalize Docker)

Reply:
Fixed.